pkg/command/plugin.go

```diff
@@ -230,22 +186,35 @@ func newDescribePluginCmd() *cobra.Command {
 	return describeCmd
 }
 
-// nolint: gocyclo
 func newInstallPluginCmd() *cobra.Command {
 	var installCmd = &cobra.Command{
 		Use:   "install [" + pluginNameCaps + "]",
 		Short: "Install a plugin",
-		Args:  cobra.MaximumNArgs(1),
+		Long:  "Install a specific plugin by name or specify all to install all plugins of a group",
+		Example: `
+    # Install all plugins of the vmware-tkg/default plugin group version v2.1.0
+    tanzu plugin install --group vmware-tkg/default:v2.1.0
+
+    # Install all plugins of the latest version of the vmware-tkg/default plugin group
+    tanzu plugin install --group vmware-tkg/default
+
+    # Install the latest version of plugin "myPlugin"
+    # If the plugin exists for more than one target, an error will be thrown
+    tanzu plugin install myPlugin
+
+    # Install the latest version of plugin "myPlugin" for target kubernetes
+    tanzu plugin install myPlugin --target k8s
+
+    # Install version v1.0.0 of plugin "myPlugin"
+    tanzu plugin install myPlugin --version v1.0.0`,
```

Comment:
Thanks for adding this. 👏

Reply:
Actually, I just moved them from lower in the code 😄 .   
Before they had to be in a `if/else` statement based on the central repo feature flag, so they couldn't be initialized in the struct itself.